src/aspire/estimation/covar2d.py

```diff
@@ -248,30 +254,38 @@ def shrink_covar_backward(self, b, b_noise, n, noise_var, shrinker):
             b_out[ell] = b_ell
         return b_out
 
-    def get_cwf_coeffs(self, coeffs, ctf_fb, ctf_idx, mean_coeff=None, covar_coeff=None, noise_var=1):
+    def get_cwf_coeffs(self, coeffs, ctf_fb=None, ctf_idx=None, mean_coeff=None, covar_coeff=None, noise_var=1):
         """
         Estimate the expansion coefficients using the Covariance Wiener Filtering (CWF) method.
 
         :param coeffs: A coefficient vector (or an array of coefficient vectors) to be calculated.
         :param ctf_fb: The CFT functions in the FB expansion.
         :param ctf_idx: An array of the CFT function indices for all 2D images.
+            If ctf_fb or ctf_idx is None, the identity filter will be applied.
         :param mean_coeff: The mean value vector from all images.
         :param covar_coeff: The block diagonal covariance matrix of the clean coefficients represented by a cell array.
-        :param noise_var: The estimated variance of noise.
+        :param noise_var: The estimated variance of noise. The value should be zero for `coeffs`
+            from clean images of simulation data.
         :return: The estimated coefficients of the unfiltered images in certain math basis.
             These are obtained using a Wiener filter with the specified covariance for the clean images
             and white noise of variance `noise_var` for the noise.
         """
         if mean_coeff is None:
             mean_coeff = self.get_mean(coeffs, ctf_fb, ctf_idx)
+        mean_coeff = mean_coeff.reshape((self.basis.count, 1))
```

Comment:
Remove.

It's better to have `mean_coeff` be a vector (that is, only have one axis) and modify it locally if other functions have different conventions. For example, below, we would have
```
mean_coeff_k = blk_diag_apply(ctf_fb_k, mean_coeff[:, np.newaxis])[:, 0]

coeff_est_k = coeff_k - mean_coeff_k[:. np.newaxis]
```
followed by
```
coeff_est_k = coeff_est_k + mean_coeff[:, np.newaxis]
```
later on. This way, the deviations from convention (i.e., that `mean_coeff` should be a vector) is clearly localized in the code.

Reply:
Done.